finalPrep/modZeroHero.js

```diff
@@ -66,19 +121,63 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
 //   - energyLevel = 50 
-
+class SuperHero {
+  constructor(name, superpower, age) {
+  this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    static archNemesis = "The Syntax Error";
+    static powerLevel = 100;
+    static energyLevel = 50;
+  }
+ }
 // - Create the following class methods
 //   - sayName, should print the hero's name to the console
 //   - maximizeEnergy, should update the energyLevel to 1000
 //   - gainPower, should take an argument of a number and INCREASE the powerLevel by that number
-
+class SuperHero {
+  constructor(name, superpower, age) {
+  this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    this.archNemesis = "The Syntax Error";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+  }
+  sayName() {
+    console.log(this.name);
+  }
+  maximizeEnergy() {
+   console.log(this.energyLevel * 20);
```

Comment:
I think the instructions are asking you to **re-assign** `this.energyLevel` to be a value of 1000, rather than calculate it.  Additionally, the method `maximizeEnergy` doesn't call for a `console.log` like `sayName` does.  

```
maximizeEnergy() {
    this.energyLevel = 1000;
}
```